src/main/java/no/nav/familie/integrasjoner/dokarkiv/DokarkivService.kt

```diff
@@ -100,7 +102,11 @@ class DokarkivService(private val dokarkivRestClient: DokarkivRestClient,
     }
 
     private fun hentNavnForFnr(fnr: String?): String {
-        return personopplysningerService.hentPersoninfoFor(fnr).navn ?: error("Kan ikke hente navn")
+        if (fnr == null) {
+            error("Kan ikke hente navn")
+        }
+
+        return personopplysningerService.hentPersoninfo(fnr, Tema.BAR.toString(), PersonInfoQuery.ENKEL).navn
```

Comment:
@bent-lorentzen @oyvindhagen @stigebil går det greit å hardkode barnetrygd her eller må det sendes inn som en parameter til controlleren?

Reply:
Det vil fungere og ikke ha så mye praktisk å si i dette tilfellet, men mest riktig ville vært å bruke riktig tema. Kunne vel hentet disse temaene fra konsument-headeren?

Reply:
Det er ingenting i veien for å gjøre det sånn. Tema brukes av PDL for å sjekke på tilgang til opplysningene det gjøres oppslag mot og det er ikke noe skille på dette for BAR, ENF og KON idag uansett.

Reply:
Kunne vi fått et "FAM"-tema eller lignende for bruk i integarsjoner?

Reply:
@oyvindhagen har du mulighet til å spørre om vi kan få et FAM tema?

Reply:
Tema er et kodeverk som brukes litt flere steder. Bl.a så styres det tilgangen til journalposter.

Reply:
Hva tenker du? At vi ikke skal innføre FAM og bare bruke BAR inntil videre?

Reply:
Saksbehandler må ha riktig ad-rolle for å se journalpost knyttet til tema. Ved å innføre nytt tema, så er det mulig å FAM blir tatt i bruk videre uten at man tar hensyn til dette. Tillegg brukes Tema til ruting etc. Så man bør ha full kontroll ved bestilling av ny.

Reply:
Ok, da beholder vi BAR inntil videre tenker jeg :)

Reply:
Akkurat her så har man Tema på metadaten. Den kan brukes mot pdl

Reply:
Det er i hvertfall ikke noe Tema i kodeverk for `familie` i dag. Om vi bare kan bestille det, eller hva som evt skal til for et en ny kode kvalifiserer til innpass i kodeverk må jeg i så fall forhøre meg om. Inntil videre kan BAR brukes som dere har gjort til nå.